easybuild/easyconfigs/f/FlashPCA2/FlashPCA2-2.0_CPPFLAGS.patch

```diff
@@ -0,0 +1,45 @@
+diff -ruN flashpca-2.0.orig/Makefile flashpca-2.0/Makefile
+--- flashpca-2.0.orig/Makefile	2016-12-15 20:33:56.000000000 -0800
++++ flashpca-2.0/Makefile	2021-08-17 17:31:26.778234000 -0700
+@@ -19,7 +19,8 @@
+    data.o \
+    util.o
+ 
+-CXXFLAGS = -I${SPECTRA_INC} -I${BOOST_INC} -I${EIGEN_INC}
++CXXFLAGS = $(CXXFLAGS)
```

Comment:
You'll need to change this to:

```make
CXXFLAGS := $(CXXFLAGS)
```

(that is, use `:=` rather than `=`)

to avoid:

```
Makefile:41: *** Recursive variable `CXXFLAGS' references itself (eventually).  Stop.
```